src/Umbraco.Infrastructure/Install/PackageMigrationRunner.cs

```diff
@@ -89,6 +90,28 @@ public IEnumerable<ExecutedMigrationPlan> RunPackageMigrationsIfPending(string p
         return RunPackagePlans(packagePlans);
     }
 
+    /// <summary>
+    ///     Checks if all executed package migrations succeeded for a package.
+    /// </summary>
+    public async Task<Attempt<bool, PackageMigrationOperationStatus>> RunningPendingPackageMigrationsSucceeded(string packageName)
```

Comment:
Maybe this should be called to something like `RunPendingPackageMigrations`? 

Currently it seems like requesting some state, but in reality you're actually running the plan.

Reply:
It will return a state for whether the migrations run successfully or not